dev/tests/js/jasmine/tests/app/code/Magento/AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/overlay.test.js

```diff
@@ -0,0 +1,283 @@
+                            data: {
+                                'form_key': 'MagentoFormkey'
+                            },
+                            dataType: 'json',
+                            context: overlayObj
+                        });
+                    promise.done(function(data, status) {
+                        handler.success(data);
+                    });
+                    promise.fail(function(status, error) {
+                        handler.fail(error);
+                    });
+                }
+            };
+
+            it('Check for defined', function () {
+                expect(overlayObj.hasOwnProperty('updateLicensed')).toBeDefined();
+            });
+
+            it('Check method type', function () {
+                var type = typeof overlayObj.updateLicensed;
+               expect(type).toEqual('function');
+            });
+
+            it("Check Ajax Success request", function() {
+
+                spyOn(handler, 'success').and.callFake(function(e) {
+                    return [1,2]
+                });
+                spyOn($, 'ajax').and.callFake(function(e) {
+                    return $.Deferred().resolve({
+                        'response': 'success'
+                    }).promise();
+                });
+                testObj.ajaxFunction();
+                expect(handler.success).toHaveBeenCalled();
+                expect(typeof handler.success()).toBe('object');
+                expect(handler.success().length).toBe(2);
+            });
+
+            it('Check Ajax failure request', function() {
+                spyOn(handler, 'fail').and.callFake(function(e) {
+                    return 'Error Message';
+                });
+                spyOn($, 'ajax').and.callFake(function(e) {
+                    return $.Deferred().reject({
+                        'response': 'failure'
+                    }).promise();
+                });
+                testObj.ajaxFunction();
+                expect(handler.fail).toHaveBeenCalled();
+                expect(typeof handler.fail()).toBe('string');
+                expect(handler.fail()).toBe('Error Message');
+            });
+        });
+
+        describe('"getIds" method', function () {
+
+            it('Check for defined ', function () {
+                expect(overlayObj.hasOwnProperty('getIds')).toBeDefined();
+            });
+
+            it('Check method type', function () {
+                var type = typeof overlayObj.getIds;
+                expect(type).toEqual('function');
+            });
+
+            it('Check returned value type if method called with object argument', function () {
+
+                overlayObj.getIds = jasmine.createSpy().and.callFake(function () {
+                    return [1,2,3];
+                });
+
+                expect(typeof overlayObj.getIds()).toEqual('object');
+            });
+
+            it('Check getIds and return array', function () {
```

Comment:
This test is not valuable as you are stubbing out the getIds directly before testing it. Either don't stub the method you are testing, or remove this test.